### PR TITLE
Feat: ecosystem update

### PR DESCRIPTION
**Pull request by moritzkirstein**
## Proposed Changes

  - add new "isLive" indicator to menu item content
  - update behavior of ecosystem homepage component
    - remove live indicator
    - remove button links
  - update behavior of menu links, remove `isLive === false` entries

### CODE REVIEW

File: src/components/Header/Menu.tsx [hunk 1]
```diff
@@ -52,8 +54,6 @@ export function MenuLink({ name, link, className }: MenuItem) {
 export default function Menu(): ReactElement {
   const { appConfig, siteContent } = useMarketMetadata()
 
-  const { setIsAutomationEnabled } = useAutomation()
-
```
Comment (Abrom8) [inline src/components/Header/Menu.tsx:56]: question: Is this intentional? Seems unrelated to the feature.

File: src/components/Header/Menu.tsx [hunk 2]
```diff
@@ -11,7 +11,6 @@ import MenuDropdown from '@components/@shared/MenuDropdown'
 import SearchButton from './SearchButton'
 import Button from '@components/@shared/atoms/Button'
 import UserPreferences from './UserPreferences'
-import { useAutomation } from '../../@context/Automation/AutomationProvider'
```
Comment (Abrom8) [inline src/components/Header/Menu.tsx:14]: question: intentional?

File: src/components/Header/Menu.tsx (hunk 1 above)
Comment (moritzkirstein) [reply src/components/Header/Menu.tsx:56]: intentional, I noticed an unused import

**Review by oceanByte — APPROVED** [review]
